tests/benchmark/compute/instruction/test_keccak.py

```diff
@@ -60,7 +61,24 @@ def test_keccak(
 
     benchmark_test(
         code_generator=JumpLoopGenerator(
-            setup=Op.PUSH20[optimal_input_length],
+            setup=Op.PUSH20(optimal_input_length),
```

Comment:
```suggestion
            setup=Op.PUSH20[optimal_input_length],
```
Original should be ok.